SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslationManager.cpp

```diff
@@ -284,9 +273,8 @@ void SpatialVirtualWorkerTranslationManager::ServerWorkerEntityQueryDelegate(con
 
 	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
-		UE_LOG(LogSpatialVirtualWorkerTranslationManager, Warning, TEXT("Server worker entity query failed: %s, retrying."),
+		UE_LOG(LogSpatialVirtualWorkerTranslationManager, Error, TEXT("Server worker entity query failed: %s, retrying."),
```

Comment:
changed this to an error because the only way this can now be achieved is when we get an error that we can't recover from